src/inventory_client/inventory_client.go

```diff
@@ -253,26 +258,46 @@ func (c *inventoryClient) GetCluster(ctx context.Context) (*models.Cluster, erro
 	return cluster.Payload, nil
 }
 
-func (c *inventoryClient) GetClusterMonitoredOperator(ctx context.Context, clusterId, operatorName string) (*models.MonitoredOperator, error) {
+func (c *inventoryClient) GetMonitoredOperator(ctx context.Context, clusterId string) (models.MonitoredOperatorsList, error) {
+	cacheKey := fmt.Sprintf("GetMonitoredOperator-%s", clusterId)
+	if val, err := c.cache.Get(cacheKey); err != ttlCache.ErrNotFound {
```

Comment:
What if err is a different error?

Reply:
NM, if it's good enough for the Go docs it's good enough here
https://pkg.go.dev/github.com/ReneKroon/ttlcache/v2#section-readme